File: LAB-1/Pr5.c
#include <stdio.h>
void main() {
    int n, count = 0;
    printf("Enter a Number to check if it is Prime or Not\n");
    scanf("%d", &n);

    for(int i = 1; i <= n; i++) {
        if(n % i == 0) {
            count++;
        }
    }

    if(count == 2) {
        printf("%d is a Prime Number\n", n);
    } else {
        printf("%d is Not a Prime Number\n", n);
    }

    return 0;
}

File: LAB-9/Pr5.java

import java.util.Scanner;

public class Pr5 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a number to check if it is prime:");
        int n = sc.nextInt();
        Pr5 p = new Pr5();
        int ans = p.getPrime(n);
        if(ans==1)
        {
            System.out.println("prime number");
        }
        else
        {
            System.out.println("not a prime number");
        }
    }
    int getPrime(int n)
    {
        for(int i=2;i<n/2;i++)
        {
            if(n%i==0)
            {
                return 0;
            }
        }
        return 1;
    }    
}
